### PR TITLE
Update procedure on HAOS installation on x86_64

### PR DESCRIPTION
**Pull request by c0ffeeca7**
## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_includes/installation/operating_system.md [hunk 1]
```diff
@@ -57,25 +57,39 @@ This guide assumes that you have a dedicated generic x86 PC (typically an Intel
 
 <b>Summary</b>
 
-You will need to write the HAOS (Home Assistant OS) disk image directly to your boot media, and then configure your x86 to boot from this media, using UEFI boot mode.
+You will need to write the HAOS (Home Assistant OS) disk image directly to your boot media, and configure your x86 to use UEFI boot mode when booting from this media.
 </div>
 
-### Configure the BIOS
+### Configure the BIOS on your x86-64 hardware
```
Comment (agners) [inline source/_includes/installation/operating_system.md:63]:
```suggestion
### Configure the BIOS on your x86-64 Hardware
```

**Review by agners — DISMISSED** [review]

File: source/_includes/installation/operating_system.md [hunk 2]
```diff
@@ -57,25 +57,39 @@ This guide assumes that you have a dedicated generic x86 PC (typically an Intel
 
 <b>Summary</b>
 
-You will need to write the HAOS (Home Assistant OS) disk image directly to your boot media, and then configure your x86 to boot from this media, using UEFI boot mode.
+You will need to write the HAOS (Home Assistant OS) disk image directly to your boot media, and configure your x86 to use UEFI boot mode when booting from this media.
 </div>
 
-### Configure the BIOS
+### Configure the BIOS on your x86-64 Hardware
```
Comment (frenck) [inline source/_includes/installation/operating_system.md]: I know this was suggested as a correction above, but it is a normal word and isn't capitalized in this context.

```suggestion
### Configure the BIOS on your x86-64 hardware
```

File: source/_includes/installation/operating_system.md [hunk 3]
```diff
@@ -57,25 +57,39 @@ This guide assumes that you have a dedicated generic x86 PC (typically an Intel
...
+1. Save the changes and exit.
 
-If you prefer to use a live operating system, follow the instructions of your Live distribution (e.g., [this Ubuntu guide](https://ubuntu.com/tutorials/try-ubuntu-before-you-install)). Once you booted the live operating system, the following steps on how to write the image to your boot media can be followed.
+- The BIOS configuration is complete.
+
+As a next step, we need to write the HA Operating System image to the target boot medium. The HAOS has no integrated installer. This means, the Operating System is not copied automatically to the internal disk.
```
Comment (frenck) [inline source/_includes/installation/operating_system.md]:
```suggestion
As a next step, we need to write the Home Assistant Operating System image to the target boot medium. The HAOS has no integrated installer. This means the Operating System is not copied automatically to the internal disk.
```

File: source/_includes/installation/operating_system.md [hunk 4]
```diff
@@ -57,25 +57,39 @@ This guide assumes that you have a dedicated generic x86 PC (typically an Intel
...
+
+- The "boot medium" is the medium your x86-64 hardware will boot from when it is running Home Assistant.
+- Typically, an internal medium is used for the x86-64 hardware. Internal media are S-ATA hard disk, S-ATA SSD, a M.2 SSD or a non-removable eMMC.
+- Alternatively, an external medium can be used to boot HAOS such as an USB SDD (not recommended).
```
Comment (frenck) [inline source/_includes/installation/operating_system.md]:
```suggestion
- The "boot medium" is the medium your x86-64 hardware will boot from when it is running Home Assistant.
- Typically, an internal medium is used for the x86-64 hardware. Examples of internal media are S-ATA hard disk, S-ATA SSD, M.2 SSD, or a non-removable eMMC.
- Alternatively, an external medium can be used to boot HAOS such as a USB SDD (not recommended).
```

File: source/_includes/installation/operating_system.md [hunk 5]
```diff
@@ -57,25 +57,39 @@ This guide assumes that you have a dedicated generic x86 PC (typically an Intel
...
+
+To install the HAOS internally on your x86-64 hardware, there are 2 methods:
+
+1. Copying the HAOS disk image from your Desktop computer onto your boot medium (e.g. by using a USB to S-ATA adapter). This is not an option for a non-removable eMMC on your x86-64 hardware, of course.
+To use this method, follow the steps described in the procedure below: [Write the image to your boot media](#write-the-image-to-your-boot-media).
+2. Copying a live operating system (e.g. Ubuntu) onto a USB device. Then, insert this USB device to your x86-64 hardware and start the Ubuntu.
```
Comment (frenck) [inline source/_includes/installation/operating_system.md]:
```suggestion
To install the HAOS internally on your x86-64 hardware, there are 2 methods:

1. Copying the HAOS disk image from your Desktop computer onto your boot medium (e.g. by using a USB to S-ATA adapter). This is not an option for a non-removable eMMC on your x86-64 hardware, of course.
To use this method, follow the steps described in the procedure below: [Write the image to your boot media](#write-the-image-to-your-boot-media).
2. Copying a live operating system (e.g. Ubuntu) onto a USB device. Then, insert this USB device into your x86-64 hardware and start the Ubuntu.
```

File: source/_includes/installation/operating_system.md [hunk 6]
```diff
@@ -134,15 +148,24 @@ _Select and copy the URL or use the "copy" button that appear when you hover it.
 ### Start up your {{site.installation.types[page.installation_type].board}}
 
 {% if page.installation_type == 'generic-x86-64' %}
-1. If you used your Desktop system to write to your boot media, install the boot media ({{site.installation.types[page.installation_type].installation_media}}) into the target system. Otherwise, shutdown the live operating system and make sure to remove the USB flash drive you have been using for the live system.
-2. Make sure an ethernet cable for network is plugged in
+
+1. If you used your Desktop system to write the HAOS your boot media, install the boot media ({{site.installation.types[page.installation_type].installation_media}}) in the generic-x86-64 system.
```
Comment (frenck) [inline source/_includes/installation/operating_system.md]:
```suggestion
1. If you used your desktop system to write the HAOS your boot media, install the boot media ({{site.installation.types[page.installation_type].installation_media}}) in the generic-x86-64 system.
```

File: source/_includes/installation/operating_system.md [hunk 7]
```diff
@@ -134,15 +148,24 @@ _Select and copy the URL or use the "copy" button that appear when you hover it.
...
+
+- If you used a live operating system (e.g. Ubuntu), shutdown the live operating system and make sure to remove the USB flash drive you used for the live system.
```
Comment (frenck) [inline source/_includes/installation/operating_system.md]:
```suggestion
- If you used a live operating system (e.g. Ubuntu), shut down the live operating system and make sure to remove the USB flash drive you used for the live system.
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]
Thanks, @c0ffeeca7 👍

**Pull request merged**